Source/WebCore/dom/Element.cpp

```diff
@@ -1984,7 +2008,15 @@ ExceptionOr<void> Element::setAttribute(const AtomString& qualifiedName, const A
     auto caseAdjustedQualifiedName = shouldIgnoreAttributeCase(*this) ? qualifiedName.convertToASCIILowercase() : qualifiedName;
     unsigned index = elementData() ? elementData()->findAttributeIndexByName(caseAdjustedQualifiedName, false) : ElementData::attributeNotFound;
     auto name = index != ElementData::attributeNotFound ? attributeAt(index).name() : QualifiedName { nullAtom(), caseAdjustedQualifiedName, nullAtom() };
-    setAttributeInternal(index, name, value, InSynchronizationOfLazyAttribute::No);
+
+    auto sink = nullString();
+    String attributeType = getAttributeTrustedType(name.localName(), getAttribute(name), ""_s, ""_s, sink);
+    auto attributeValue = getTrustedTypesCompliantAttributeValue(attributeType, value, this, sink);
```

Comment:
Change this sink param to just be "Element setAttribute"_s

I know this doesn't match the specification but the spec is wrong (https://github.com/w3c/trusted-types/issues/492)